src/main/java/minicraft/level/tile/farming/PlantTile.java

```diff
@@ -74,6 +80,6 @@ protected void harvest(Level level, int x, int y, Entity entity) {
 		// Play sound.
 		Sound.play("monsterhurt");
 
-        level.setTile(x, y, Tiles.get("Dirt"));
+        level.setTile(x, y, Tiles.get("Farmland"));
```

Comment:
I would like to keep the original mechanics.

Reply:
I guess this does affect when `PlantTile.hurt` is called. Alright.